connector/spi/core-spi/src/main/java/org/oaebudt/edc/spi/store/ParticipantGroupStore.java

```diff
@@ -0,0 +1,13 @@
+package org.oaebudt.edc.spi.store;
+
+import java.util.Map;
+import java.util.Set;
+
+public interface ParticipantGroupStore {
```

Comment:
I'd suggest to model the group as a class/record with `id` and `participants` properties, and have standard `findById`, `save`, `getAll` store methods, and leave the overlying service to manage the logic